dbos_transact/dbos.py

```diff
@@ -84,34 +91,16 @@
 TEMP_SEND_WF_NAME = "<temp>.temp_send_workflow"
 
 
-class WorkflowProtocol(Protocol[P, R]):
+class DBOSCallProtocol(Protocol[R]):
```

Comment:
Without using the parameter generics, is `startWorkflow` still properly type-checked?